app/main.py

```diff
@@ -1,3 +1,67 @@
+from __future__ import annotations
+
+
 class Distance:
-    # Write your code here
-    pass
+
+    def __init__(self, km: float) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: (Distance | int | float)) -> Distance:
+        if isinstance(other, Distance):
+            return Distance(self.km + other.km)
+        elif isinstance(other, float) or isinstance(other, int):
+            return Distance(self.km + other)
+        else:
+            return self
+
+    def __iadd__(self, other: (Distance | int | float)) -> Distance:
+        if isinstance(other, Distance):
+            self.km = self.km + other.km
+        if isinstance(other, float) or isinstance(other, int):
+            self.km = self.km + other
+        return self
+
+    def __mul__(self, num: float) -> Distance:
+        return Distance(self.km * num)
+
+    def __truediv__(self, num: float) -> Distance:
+        if num != 0:
+            return Distance(round(self.km / num, 2))
+        else:
+            raise ZeroDivisionError(f"num = {num} zero division error")
+
+    def __lt__(self, other: (Distance | int | float)) -> bool:
+        if isinstance(other, Distance):
+            return self.km < other.km
+        if isinstance(other, float) or isinstance(other, int):
```

Comment:
Simplify this code here and in the other places
```suggestion
        if isinstance(other, (float, int)):
```